README.md

```diff
@@ -0,0 +1,181 @@
+# Needly (Version 2.5)
```

Comment:
no user will see this README

Reply:
Which means I need to do something to make it visible? How? Or it does not make sense to include the README.md in the package?

Reply:
It doesn't make sense to include it here. You should add it to your GitHub repo instead.